oma_openapi_library.json

```diff
@@ -66,12 +66,12 @@
           }
         }
       },
-      "/organizations": {
+      "/publishers": {

```

Comment:
@jmudge  I believe that I made a mistake by calling this element (organizations) in the plural. As we are not planning to refer to any document that it is not OMA. I will check it a later stage.